etc/config/c++.win32.properties

```diff
@@ -1 +1,38 @@
-supportsBinary=true
+# Default settings for C++ on Windows
+
+compilers=&vc2017:&clang
+postProcess=
+demanglerClassFile=./demangler-win32
+objdumper=objdump
+options=
+supportsBinary=false
+stubRe=\bmain\b
+stubText=int main(void){return 0;/*stub provided by Compiler Explorer*/}
+
+
+# clang compilers
+
+group.clang.compilers=&clang_32:&clang_64
+group.clang.intelAsm=-mllvm --x86-asm-syntax=intel
+group.clang.groupName=clang
+group.clang.compilerType=win32
+group.clang.binaryHideFuncRe=^(__.*|_(init|start|fini)|(de)?register_tm_clones|call_gmon_start|frame_dummy|\.plt.*)$
+
+group.clang_32.groupName=clang x86
+group.clang_32.options=-m32
+group.clang_64.groupName=clang amd64
+
+
+# visual C++ 2017 compilers
+
+group.vc2017.compilers=&vc2017_32:&vc2017_64
+group.vc2017.options=-EHsc
+group.vc2017.compilerType=win32-vc
```

Comment:
I did a silly thing where I used WSL to clone the repo, but ran this under Windows, and because the file is called "lib/compilers/win32-VC.js" it's not found. I suggest renaming win32-VC.js to win32-vc.js (all lowercase) just in case of this happening to others.

Reply:
(or to Win32-VC and rename it in this file)